src/components/LoginForm/index.js

```diff
@@ -186,22 +169,47 @@ const LoginFormUI = (props) => {
                   })}
                   onChange={(e) => hanldeChangeInput(e)}
                 />
+                <ForgotPassword>
+                  {t('FORGOT_YOUR_PASSWORD', 'Forgot your password?')} {elementLinkToForgotPassword}
+                </ForgotPassword>
                 <Button color='primary' type='submit' disabled={formState.loading}>
                   {formState.loading ? t('LOADING') + '...' : t('LOGIN')}
                 </Button>
               </FormInput>
             )}
         </>
-
-        <ForgotPassword>
-          {t('FORGOT_YOUR_PASSWORD', 'Forgot your password?')} {elementLinkToForgotPassword}
-        </ForgotPassword>
         {linkToForgetPassword && (
           <>
             {t('NEW_ON_PLATFORM')}
-            <a href={linkToForgetPassword}>{t('RESET_PASSWORD','Reset Password')}</a>
+            <a href={linkToForgetPassword}>{t('RESET_PASSWORD', 'Reset Password')}</a>
           </>
         )}
+        <NewOnPlatform>
+          {elementLinkToSignup && (
+            <>
+              {t('NEW_ON_PLATFORM', 'New on Ordering?')} {elementLinkToSignup}
+            </>
+          )}
+          {linkToSignup && (
+            <>
+              {t('NEW_ON_PLATFORM', 'New on Ordering?')}
+              <a href={linkToSignup}>{t('CREATE_AN_ACCOUNT', 'Create an account')}</a>
+            </>
+          )}
+        </NewOnPlatform>
+        {/* <SocialIcons>
+          {configs?.facebook_id && <FacebookLoginButton ordering={ordering} appId={configs.facebook_id.value} handleSuccessFacebookLogin={handleSuccessFacebook} />}
+          {/* <FaApple />
+                  <AiOutlineGoogle />
+        </SocialIcons> */}

```

Comment:
Why this comment?

Reply:
This is for the mobile design

Reply:
ok, if it's unused code, please delete, in another case, add comment description with `/** --comment-- */` notation.